app/src/main/java/com/lab/skuld/ui/screens/NewNoteScreen.kt

```diff
@@ -143,11 +156,18 @@ fun ShowNewNoteScreen(documentt: Event = emptyEventorTask){
 
 
 
-
+    val focusManager = LocalFocusManager.current
```

Comment:
This should be removed, pull from master and test if App.kt:208 implementation of this handles your use-case, but I suspect it does

Reply:
Removed, your implementation handles this use-case too.